x/market/client/cli/tx.go

```diff
@@ -210,7 +210,7 @@ func cmdLeaseWithdraw(key string) *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
-	AddLeaseIDFlags(cmd.Flags(), dcli.DeploymentIDOptionNoOwner(true))
```

Comment:
Didn't we just add these configuration options @troian?

Reply:
In this case they should be used by `Req` but not `Add`